### PR TITLE
Added prometheus operator

### PR DESCRIPTION
**Pull request by robshelly**

### CODE REVIEW

**Comment by robshelly** [comment]
Made the changes suggested including breaking the monitoring resources into the 3 categories of Infrastructure, Service Monitors and Alerting.

However, I think in the future we might decide to combine the Service Monitor and Alerting, as the alerting resources includes all Prometheus rules not specific to alerting. Currently we don't have any non alerting rules but I think we might in the future.

File: documentation/modules/proc-deploy-monitoring-infrastructure.adoc [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+[options="nowrap",subs="+quotes,attributes"]
+----
+{cmdcli} project _{ProductNamespace}_
+----
+
+. Deploy using the `monitoring-infrastructure` bundle.
++
+[options="nowrap",subs="attributes"]
+----
+{cmdcli} apply -f install/bundles/monitoring-infrastructure
```
Comment (lulf) [inline documentation/modules/proc-deploy-monitoring-infrastructure.adoc]: Should this be 'monitoring' ?

Comment (robshelly) [reply documentation/modules/proc-deploy-monitoring-infrastructure.adoc]: Yes it should be 'monitoring'. Fixed!

**Comment by lulf** [comment]
@enmasse-ci run tests

File: documentation/modules/proc-deploy-service-monitors.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// assembly-monitoring-kube.adoc
+// assembly-monitoring-oc.adoc
+
+[id='deploy-service-monitors-{context}']
+= Deploying Service Monitors
+
+Follows these steps to enable Prometheus to monitor {ProductName}.
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-service-monitors.adoc]: Change to: You must deploy the service monitors to enable Prometheus to monitor {ProductName}.

File: documentation/modules/proc-deploy-service-monitors.adoc [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// assembly-monitoring-kube.adoc
+// assembly-monitoring-oc.adoc
+
+[id='deploy-service-monitors-{context}']
+= Deploying Service Monitors
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-service-monitors.adoc]: lowercase Service Monitors

File: documentation/modules/proc-deploy-service-monitors.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+Follows these steps to enable Prometheus to monitor {ProductName}.
+
+.Prerequisites
+
+* Monitoring infrastructure must be present, either pre-existing or installed using the above steps. 
+
+.Procedure
+
+. Select the {ProductName} namespace.
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-service-monitors.adoc]: Change period to a colon

File: documentation/modules/proc-deploy-service-monitors.adoc [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
++
+[options="nowrap",subs="+quotes,attributes"]
+----
+{cmdcli} project _{ProductNamespace}_
+----
+
+. Deploy the `service-monitors`.
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-service-monitors.adoc]: Change period to a colon

File: documentation/modules/proc-deploy-service-monitors.adoc [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+// assembly-monitoring-oc.adoc
+
+[id='deploy-service-monitors-{context}']
+= Deploying Service Monitors
+
+Follows these steps to enable Prometheus to monitor {ProductName}.
+
+.Prerequisites
+
+* Monitoring infrastructure must be present, either pre-existing or installed using the above steps.
```
Comment (jenmalloy) [inline documentation/modules/proc-deploy-service-monitors.adoc]: Change to: Monitoring infrastructure must be present, whether pre-existing or link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#deploy-monitoring-infrastructure-messaging[installed following this procedure].

File: documentation/modules/proc-installing-using-ansible.adoc [hunk 7]
```diff
@@ -30,8 +30,10 @@ The following Ansible configuration settings are supported:
 |api_server|Specifies whether to enable the REST API server|True |no
 |register_api_server|Specifies whether to register the API server with {KubePlatform} master|False |no
 |secure_api_server|Specifies whether to enable mutual TLS for the API server|False |no
-|install_example_plans|Specifies whether to install example plans and infrastructure configs|True|no|
-|enable_monitoring|Specifies whether to enable the monitoring services|Not applicable|no
+|install_example_plans|Specifies whether to install example plans and infrastructure configs|True|no
```
Comment (jenmalloy) [inline documentation/modules/proc-installing-using-ansible.adoc]: Change "configs" to "configurations"

File: documentation/modules/proc-installing-using-ansible.adoc [hunk 8]
```diff
@@ -30,8 +30,10 @@ The following Ansible configuration settings are supported:
...
+|monitoring_infrastructure|Specifies whether to installs full monitoring infrastructure|Not applicable|no
+|service_monitors|Specifies whether to install the service monitors for monitoing {ProductName}|Not applicable|no
```
Comment (jenmalloy) [inline documentation/modules/proc-installing-using-ansible.adoc]: monitoring (missing the r)

File: documentation/modules/proc-installing-using-ansible.adoc [hunk 9]
```diff
@@ -30,8 +30,10 @@ The following Ansible configuration settings are supported:
...
+|alerting|Specifies whether to install Prometheus rules which configure alerting for {ProductName}|Not applicable|no
```
Comment (jenmalloy) [inline documentation/modules/proc-installing-using-ansible.adoc]: Change "which" to "that"

**Pull request merged**